src/puppetlabs/trapperkeeper/services/metrics/metrics_core.clj

```diff
@@ -22,10 +24,15 @@
 (def ReportersConfig
   {(schema/optional-key :jmx) JmxReporterConfig})
 
+(def JolokiaApiConfig
+  {(schema/optional-key :enabled) schema/Bool
+   (schema/optional-key :config) jolokia/JolokiaConfig})
+
 (def MetricsConfig
   {:server-id                       schema/Str
    (schema/optional-key :enabled)   schema/Bool
-   (schema/optional-key :reporters) ReportersConfig})
+   (schema/optional-key :reporters) ReportersConfig
+   (schema/optional-key :jolokia) JolokiaApiConfig})
```

Comment:
In pe-tk-metrics we've ended up making reporter config per-registry, rather than global (for all registries), with the hope that ultimately we would change the config here. The ticket for that is TK-393. I don't want to block this PR on that though - just wanted to mention it cuz we're probably going to have to do some additional hackery in pe-tk-metrics to support this or we'll need to do TK-393 sooner rather than later.